api/envoy/admin/v3/config_dump.proto

```diff
@@ -49,13 +49,17 @@ message UpdateFailureState {
       "envoy.admin.v2alpha.UpdateFailureState";
 
   // What the component configuration would have been if the update had succeeded.
+  // This field may not be populated by xDS clients due to storage overhead.
   google.protobuf.Any failed_configuration = 1;
 
   // Time of the latest failed update attempt.
   google.protobuf.Timestamp last_update_attempt = 2;
 
   // Details about the last failed update attempt.
   string details = 3;
+
+  // This is the version of the rejected resource.
+  string version_info = 4;
```

Comment:
I think this also needs not-implemented-hide.

Reply:
Done.